.eslintrc

```diff
@@ -1,62 +1,60 @@
 {
   "plugins": ["prettier"],
+  "extends": "eslint:recommended",
   "env": {
 	"node": true
   },
   "parserOptions": {
 	"ecmaVersion": 2015
   },
   "rules": {
-	"no-underscore-dangle": [
-	  0
-	],
-	"no-mixed-requires": [
-	  0
-	],
-	"new-cap": [
-	  0
-	],
-	"no-console": 2,
-	"no-debugger": 2,
-	"no-path-concat": 0,
-	"no-shadow": 1,
-	"no-array-constructor": 2,
-	"no-caller": 2,
-	"no-eval": 2,
-	"no-extend-native": 2,
-	"no-extra-bind": 2,
-	"no-with": 2,
-	"no-loop-func": 2,
-	"no-multi-str": 2,
-	"no-new-func": 2,
-	"no-new-object": 2,
-	"no-return-assign": 2,
-	"no-sequences": 2,
-	"no-shadow-restricted-names": 2,
-	"no-unused-expressions": 2,
-	"no-unused-vars": 2,
-	"no-use-before-define": 0,
-	"no-new": 0,
-	"dot-notation": [
-	  2,
-	  {
-		"allowKeywords": true
+	"no-underscore-dangle": "off",
```

Comment:
Im guessing it's just a copy and paste from another project I used, so no reason really :)